testgres/node.py

```diff
@@ -127,20 +127,23 @@ def __repr__(self):
 
 
 class PostgresNode(object):
-    def __init__(self, name=None, port=None, base_dir=None, conn_params: ConnectionParams = ConnectionParams()):
+    def __init__(self, name=None, port=None, base_dir=None, conn_params: ConnectionParams = ConnectionParams(), bin_dir=None, prefix=None):
```

Comment:
Do you really need to change anything with respect to `prefix`? It only determines the path to the temporary folders used for running tests. Tests are run with the new node only, not with the old one, so the old node should not need temporary folders (and thus its own `prefix`) at all.

Reply:
Prefix is used as parameter for this structure
`self._base_dir = self.os_ops.mkdtemp(prefix=self._prefix or TMP_NODE)`
TMP_NODE - is just generated string that hard to read and memorize. Adding the parameter will make tests easer for reading and understanding.